tests/trading/test_trade.py

```diff
@@ -559,7 +554,7 @@ def test_one_ask_on_books_buy_excess_order(contractsFixture):
             "_event_type": "MakeOrder",
             "market": market.address,
             "outcome": YES,
-            "orderType": BUY,
+            "orderType": LONG,
```

Comment:
This was wrong before, it should have been BID for the order type, and still be BID.  Orders have type (BID/ASK) participants have direction (SHORT/LONG).